.github/workflows/master.yaml

```diff
@@ -90,13 +90,21 @@ jobs:
           AC_IDENTITY_P12: ${{ secrets.AC_IDENTITY_P12 }}
       - name: Push Docker images
         run: |
-          docker tag "${IMG}" "${IMG%%:*}:${IMAGE_TAG}"
-          docker tag "${REDSKYCTL_IMG}" "${REDSKYCTL_IMG%%:*}:${IMAGE_TAG}"
-          docker tag "${SETUPTOOLS_IMG}" "${SETUPTOOLS_IMG%%:*}:${IMAGE_TAG}"
           make docker-push
-          docker push "${IMG%%:*}:${IMAGE_TAG}"
-          docker push "${REDSKYCTL_IMG%%:*}:${IMAGE_TAG}"
-          docker push "${SETUPTOOLS_IMG%%:*}:${IMAGE_TAG}"
+      - name: Push additional Docker tags
+        run: |
+          docker-push() {
+            docker tag "$1" "$2"
+            docker push "$2"
+          }
+          docker-push "${IMG}" "${IMG%%:*}:${IMAGE_TAG}"
+          docker-push "${REDSKYCTL_IMG}" "${REDSKYCTL_IMG%%:*}:${IMAGE_TAG}"
+          docker-push "${SETUPTOOLS_IMG}" "${SETUPTOOLS_IMG%%:*}:${IMAGE_TAG}"
```

Comment:
Should we drop these in favor of the `make docker-push` above, or drop the `Push Docker Images` step?

Reply:
I was trying to break it into steps to avoid documentation; the "Push Docker images" pushes the images used by the product and the "Push additional Docker tags" just pushes the extra tags (e.g. `latest`/`edge`). Though I guess it also pushes to the dev registry so maybe the name wasn't the best...

Reply:
I went back to the one "Push Docker images" and made them the same for master and PRs (master has the extra step of forcing a GitHub Container Registry push).